graphalytics-core/src/main/java/nl/tudelft/graphalytics/validation/rule/EpsilonValidationRule.java

```diff
@@ -0,0 +1,35 @@
+package nl.tudelft.graphalytics.validation.rule;
+
+public class EpsilonValidationRule implements ValidationRule<Double> {
+
+	private final static double COMPARISON_THRESHOLD = 0.0001;
+
+	@Override
+	public Double parse(String val) {
+		try {
+			return Double.parseDouble(val);
+		} catch(NumberFormatException e) {
+			return null;
+		}
+	}
+
+	@Override
+	public boolean match(Double outputValue, Double correctValue) {
+		try {
+			double a = outputValue;
+			double b = correctValue;
+
+			if (Double.isNaN(a) && Double.isNaN(b)) {
+				return true;
+			} else if (Double.isInfinite(a) && Double.isInfinite(b) && a * b < 0) {
```

Comment:
Are two infinite values not comparable using `==`?

I believe the multiplication should yield a positive number if the signs are equal (`a * b > 0`). Better yet, check the signs more explicitly (I assume this works for pos/neg infinity): `Math.signum(a) == Math.signum(b)`.